lib/y2lan_restart_common.pm

```diff
@@ -150,12 +150,13 @@ sub check_network_status {
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Do not expect Network restart/reload. Journalctl cant find events");
     }
     else {
-        assert_equals($expected_status, 'restart', "Network restart occurred, even though it wasn't expected.");
+        my $journal_findings = script_output('cat journal.log');
+        assert_equals($expected_status, 'reload', "Expect Network restart/reload. Journalctl found \n$journal_findings\n\n");
```

Comment:
isn't missing some else here? you replace the restart check by reload, so for detecting restart now...?

Reply:
fixed in following commits. see comments above